packages/common/src/authentication/api/api.ts

```diff
@@ -72,16 +64,18 @@ export const getAuthQueries = (sdk: Sdk) => ({
       const result = await sdk.refreshToken();
       return refreshTokenGuard(result);
     },
-    stores:
-      ({ filterBy, first, offset }: ListParams) =>
-      async () => {
-        const result = await sdk.stores({
-          filter: filterBy,
-          first,
-          offset,
-        });
-
-        return result.stores;
-      },
+    me: async (token?: string) => {
+      const result = await sdk.me(
+        {},
+        {
```

Comment:
manually setting the header here, to ensure the correct token.
Waiting for the gql context to update state sometimes doesn't work, and this is more reliable